### PR TITLE
update openSUSE repos

### PR DESCRIPTION
**Pull request by denisok**
update repos for openSUSE.

### CODE REVIEW

**Comment by denisok** [comment]
need further testing on my end.

**Comment by jschmid1** [comment]
@denisok Just ping me if this is ready to be reviewed

**Comment by denisok** [comment]
@jschmid1 ready - please review. It doesn't really work with files section. I will open couple of issues.

File: openSUSE_vagrant_setup.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+zypper in -y ruby-devel
+zypper in -y gcc gcc-c++ make
+zypper in -y qemu-kvm libvirt-daemon-qemu libvirt libvirt-devel
+
+#need for vagrant-libvirt
+gem install ffi -v '1.9.18'
+gem install unf_ext -v '0.0.7.4'
+gem install ruby-libvirt -v '0.7.0'
+
+systemctl start libvirtd
```
Comment (jschmid1) [inline openSUSE_vagrant_setup.sh]: maybe also enable.

**Comment by denisok** [comment]
@jschmid1 that is a good idea to use symlinks, the question is only what should be a source for symlink.

I propose node directory will be a source and then openSUSE directories and SLES after it.

Or probably better to have separate dir with different types of configs:
like files/salt/configs/[tiny, default ...]
and symlink everything on those

What do you think? And maybe it is a next step of refactoring, goal of this PR is just to add openSUSE.

File: openSUSE_vagrant_setup.sh (hunk 1 above)
Comment (denisok) [reply openSUSE_vagrant_setup.sh]: addressed

**Pull request merged**